src/Cns.test.ts

```diff
@@ -0,0 +1,91 @@
+const labelDomain = "label.crypto";
```

Comment:
It is weird to have a variable defined before `import`. Lets move it after `import`.